@navikt/core/react/src/date/utils/check-dates.ts

```diff
@@ -9,5 +9,5 @@ export const dateIsInCurrentMonth = (
 
 /** @private */
 export function isValidDate(day: Date): boolean {
-  return day && !isNaN(day?.getTime()) && day.getFullYear() > 999;
+  return day && !Number.isNaN(day?.getTime()) && day.getFullYear() > 999;
```

Comment:
Since you check that `day` is truthy first, the question mark is redundant.
If you expect that `day` might be something else than a `Date`, then maybe the type should reflect that. If not, maybe the truthyness check can be removed.

Reply:
true! could update the function paramemter type here :) (it's already of type `day: Date` meaning all calls to this function should pass in a valid Date (and it'll probably always be truthy? I haven't actually deep dived into `Date` 😅 )